features/all-positions/AllSponsors.tsx

```diff
@@ -162,85 +194,119 @@ const AllSponsors = () => {
+                    </strong>
                   </TableCell>
-                </TableRow>
-              );
-            })}
-          </TableBody>
-        </Table>
-        <PageButtons>
-          <div
-            onClick={() => {
-              setPage(page === 1 ? page : page - 1);
-            }}
-          >
-            <Arrow faded={page === 1 ? true : false}>←</Arrow>
-          </div>
-          {"Page " + page + " of " + maxPage}
-          <div
-            onClick={() => {
-              setPage(page === maxPage ? page : page + 1);
-            }}
-          >
-            <Arrow faded={page === maxPage ? true : false}>→</Arrow>
-          </div>
-        </PageButtons>
-      </TableContainer>
+                </Tooltip>
+                <TableCell align="right"></TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {reformattedSponsorData.map((sponsor: string) => {
+                const activeSponsor = activeSponsors[sponsor];
+                return (
+                  <StyledTableRow key={sponsor}>
+                    <TableCell component="th" scope="row">
+                      <a href={getEtherscanUrl(sponsor)} target="_blank">
+                        {prettyAddress(sponsor)}
+                      </a>
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(Number(activeSponsor.collateral))}
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(Number(activeSponsor.tokensOutstanding))}
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(Number(activeSponsor.cRatio))}
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(Number(activeSponsor.liquidationPrice))}
+                    </TableCell>
+                    <TableCell align="right">
+                      <MoreInfo
+                        onClick={() => handleOpenActionsDialog(sponsor)}
+                      >
+                        ⋮
+                      </MoreInfo>
+                    </TableCell>
+                  </StyledTableRow>
+                );
+              })}
+            </TableBody>
+          </Table>
+          <PageButtons>
+            <div
+              onClick={() => {
+                setPage(page === 1 ? page : page - 1);
+              }}
+            >
+              <Arrow faded={page === 1 ? true : false}>←</Arrow>
+            </div>
+            {"Page " + page + " of " + maxPage}
+            <div
+              onClick={() => {
+                setPage(page === maxPage ? page : page + 1);
+              }}
+            >
+              <Arrow faded={page === maxPage ? true : false}>→</Arrow>
+            </div>
+          </PageButtons>
+        </TableContainer>
+        <PositionActionsDialog
```

Comment:
You should not render this actions button if `empState.isExpired`

Reply:
I've chosen to still render the button as it shows additional position info. Rather, I simply hide the liquidation and dispute logic in the dialog if expired.